### DISCUSSION TITLE
Download by model hash?

### DISCUSSION THREAD

**Discussion by bbdbby** [Confuzu/CivitAI_Image_grabber#2] [Ideas]
This program is amazing! Thank you so much for making it! I am curious, does the CivitAI API allow one to download image by model hash? After this user downloader, that would be my next dream program.

**Comment by Confuzu** [comment]
if i understand you  right you want to  download all images from a specific Model across all users? For Example all images the uses the DreamShaper Model.  If so yes the API  allows it.  But it would take time. I don't have much right now.

**Reply by bbdbby** [reply]
Ok, thanks for the response!

**Reply by Confuzu** [reply]
update is online
